spec/views/registered_applications_controller/update.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "registered_applications_controller/update.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Many unused *_spec.rb files in this. Try to keep these skeleton specs out of commits to keep codebase tidier and PRs focused (and selfishly for me it keeps PRs easier to review!). Feel free to delete the unused specs.